detox/src/devices/drivers/android/genycloud/exec/GenyCloudExec.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash');
+const exec = require('../../../../../utils/exec').execWithRetriesAndLogs;
+
+class GenyCloudExec {
+  getRecipe(name) {
```

Comment:
I'd suggest making `{ name }` an optional parameter.

Reply:
... then we would have a `getReceipes` method